cmd/console/styles/console.css

```diff
@@ -324,6 +319,33 @@ table th.icon {
 	width: 7rem;
 }
 
+table.navigation tbody tr {
+	cursor: pointer;
+}
+
+table.navigation tbody tr:hover {
+	background: rgb(236, 240, 241);
+}
+
+ul.actions {
+	display: flex;
```

Comment:
@xla Not using CSS Grids, thats quite sad...
Nimm jetzt sofort deine Hand runter und tuh mir nicht mehr weh!

@nilsen340 Please make sure he keeps his hands low.